src/Utils/openAuthModal.ts

```diff
@@ -19,7 +19,13 @@ export enum AuthModalIntent {
   SaveArtwork = "save artwork",
 }
 
-export const openAuthModal = (
+export const openAuthModal = (mediator: Mediator, options: ModalOptions) => {
+  mediator.trigger("open:auth", {
+    ...options,
+  })
```

Comment:
#minor - no need to create an intermediate object here:
```js
mediator.trigger("open:auth", options)
```

Reply:
@eessex - As a side architectural thought, do you have any opinions about possibly plucking `mediator` from window? My thinking is:

```js
// force -- global_client_setup.tsx
import mediator from 'lib/mediator'
...
window.mediator = mediator // or some other window.varName, etc
```

Then in Reaction, inside of these helpers, we can grab mediator from there, rather than having to thread it via props throughout the app and pass it in as a parameter at each call-site. This would  then allow us to funnel all mediator references into a single location, and remove a lot of code in general. Since mediator can only ever be called on the client, this is safe.

Reply:
I think this would be helpful as a next step, we are calling on `mediator` via reaction with a variety of strategies. Once we've got the helper in place it will be easier to manage this change as a separate work item.